Assets/HotmaxEditorVR/loadPropsMenu.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.Linq;
+
+public class loadPropsMenu : MonoBehaviour
+{
+    //--local refs
+    private stateManager _stateManagerMutatorRef;
+    private stateManager.editorModes _editorMode = stateManager.editorMode;
+    private bool _playerIsLocomoting = stateManager.playerIsLocomoting;
+
+    //--child gameObject refs
+    public GameObject propTiles;
+    
+    //--Lists to handle getting unique props in menu
+    private List<string> propName = new List<string>();
```

Comment:
making your variable names plural if they are a collection is a good habit. that way you can do `propName in propNames` or whatever in your loops.

Reply:
also just clearer in other parts of your code.

Reply:
got it